### PR TITLE
docs: update troubleshooting

### PR DESCRIPTION
**Pull request by Plamen5kov**

### CODE REVIEW

**Review by etabakov — COMMENTED** [review]
Let's keep the credits, but copy the content to avoid users switching to different website.

**Comment by Plamen5kov** [comment]
ping @etabakov
  https://github.com/NativeScript/docs/blob/5df196e734b59046d13112ac44b7539f5205861d/docs/start/troubleshooting.md

File: docs/start/troubleshooting.md [hunk 1]
```diff
@@ -82,3 +82,112 @@ The package manager service found that the device didn't have enough storage spa
...
+* tns doctor not finding any problems but tns run android --timeout 300 times out with error.
+* Command gradlew.bat failed with exit code 1.
+* `tns run android` throws Java (javac) error.
+
+**Solution:**
+
+### 1. Black screen when running AVD emulator
+
+#### Problem & Suspected reason
+When I was trying to run an android emulator using AVD Manager, as Administrator, I was getting a back screen. The reason was that I did not have all the necessary Android SDK packets. This might be because I already had Visual Studio & c# Xamarin extension installed that after running `@powershell -NoProfile -ExecutionPolicy Bypass -Command "iex ((new-object net.webclient).DownloadString('https://www.nativescript.org/setup/win'))"` confused the hell out of my Android emulator.
```
Comment (etabakov) [inline docs/start/troubleshooting.md:101]: The style is very different from everything else in the documentation, using "I" and "My" doesn't comply to our style guide. We should either indicate that the whole content is a quote of a community member (by using the quote markdown) or rephrase it to match our style guide.

File: docs/start/troubleshooting.md [hunk 2]
```diff
@@ -82,3 +82,112 @@ The package manager service found that the device didn't have enough storage spa
...
+![AVD manager create](34569623-3de7cae2-f161-11e7-89aa-dd5d190a1ca5.JPG)
+
+**Note:** I have plenty of RAM so I left the default RAM amount, but you might need to edit that for your specific system. That being said, I increase VM Heap to 512 because the virtual machine was not loading. 
+8. Click on Ok > Start...
+9. Profit!
+
+### 2. AVD Android emulator is laggy when running
+
+#### Problem & Suspected reason
+My uneducated assumption in my case the problem was forcing my old CPU I5-3750k to calculate polygons which the processor is not very successful at.
```
Comment (etabakov) [inline docs/start/troubleshooting.md:128]: Same as above.

File: docs/start/troubleshooting.md [hunk 3]
```diff
@@ -82,3 +82,112 @@ The package manager service found that the device didn't have enough storage spa
...
+**Note:** At this point you might find that the NativeScript is running to a **"Build Error"** and it is exiting. Now... that is documented here #2092. I found that creating a fresh TNS project works best than amending package.json, but you are welcome to do either.
+![build_error](34571332-19a317ae-f166-11e7-828a-fd50319cc474.JPG)
+4. Profit!
+![successfull build](34571454-86c04244-f166-11e7-8856-bd67ca8a2f6f.png)
+
+
+### 4. Command gradlew.bat failed with exit code 1.
+
+#### Problem & Suspected reason
+As explained by @Plamen5kov in ticket #2092 the problem is due to conflicting jars.
```
Comment (etabakov) [inline docs/start/troubleshooting.md]: Let's provide link to the issue here.

File: docs/start/troubleshooting.md [hunk 4]
```diff
@@ -82,3 +82,112 @@ The package manager service found that the device didn't have enough storage spa
...
+
+#### Fix
+Easiest fix is to build a new app from scratch and port your scripts. alternatively you can follow the steps from ticket #2092 by @Pip3r4o
```
Comment (etabakov) [inline docs/start/troubleshooting.md]: Same as above.

**Review by etabakov — APPROVED** [review]
